.puppet-lint.rc

```diff
@@ -0,0 +1,3 @@
+--fail-on-warnings
+--no-parameter_documentation-check
```

Comment:
it looks like we can opt-in to those two? In that case add the following to the .sync.yml
```
---
.puppet-lint.rc:
  enabled_lint_checks:
    - parameter_documentation
    - parameter_types
```